services/auth/test/scoperesolver_test.js

```diff
@@ -2,15 +2,14 @@ const helper = require('./helper');
 const ScopeResolver = require('../src/scoperesolver');
 const exchanges = require('../src/exchanges');
 const {scopeCompare} = require('taskcluster-lib-scopes');
-const MonitorManager = require('taskcluster-lib-monitor');
+const monitorManager = require('../src/monitor');
```

Comment:
it's the same as was before, isn't it? 🤔

Reply:
Not quite. The `src/monitor` does some setup to lib-monitor first.